pkg/handlers/generic/mutation/mirrors/containerd_files.go

```diff
@@ -76,7 +69,7 @@ func (c containerdConfig) needContainerdConfiguration() bool {
 //     https://github.com/containerd/containerd/blob/main/docs/hosts.md#setup-default-mirror-for-all-registries
 //
 //  2. Setting CA certificate for global image registry mirror and image registries.
-func generateContainerdHostsFile(
+func generateContainerdDefaultHostsFile(
```

Comment:
Does the godoc needs to be updated?

Reply:
Removed part about configuring CA certs for non-mirror registries.

Reply:
Thank you!